.bazelrc

```diff
@@ -77,18 +76,18 @@ build:workflows --flaky_test_attempts=//enterprise/server/test/webdriver/invocat
 build:workflows --config=buildbuddy_bes_backend
 build:workflows --config=buildbuddy_bes_results_url
 build:workflows --config=buildbuddy_remote_cache
-build:workflows --config=buildbuddy_remote_executor
 build:workflows --config=buildbuddy_experimental_remote_downloader
 build:workflows --@io_bazel_rules_go//go/config:race
 
+# Configuration used for Linux workflows
+build:linux-workflows --config=remote-shared
+build:linux-workflows --config=workflows
+build:linux-workflows --config=buildbuddy_remote_executor
+
+# Configuration used for Mac workflows
 # TODO(bduffany): Enable RBE for Mac workflows, and reconcile this with other configs
 build:mac-workflows --config=cache
-build:mac-workflows --remote_instance_name=buildbuddy-io/buildbuddy/workflows
-build:mac-workflows --experimental_remote_cache_compression
```

Comment:
did this get dropped? Trying to find where it gets pulled in as part of workflows but not sure it's there (or maybe i'm missing it)

Reply:
experimental_remote_cache_compression is already set in `--config=cache` so it should be safe to remove